docs/guides/advanced/using_read_tokens.md

```diff
@@ -0,0 +1,196 @@
+In addition to [write tokens](./creating_write_tokens.md), Logfire also supports **read tokens**.
+A read token allows you to query data from your Logfire project using our API.
+
+## What are Read Tokens?
+
+Read tokens provide secure, programmatic access to the data in your Logfire project. They can be used to run arbitrary
+SQL queries against your data, offering flexibility and power for data analysis and integration with other tools.
+
+If you've set up Logfire following the [first steps guide](../first_steps/index.md), you can generate read tokens from
+the Logfire web interface to start querying your data.
+
+## How to Create a Read Token
+
+1. Open the **Logfire** web interface at [logfire.pydantic.dev](https://logfire.pydantic.dev).
+2. Select your project from the **Projects** section on the left-hand side of the page.
+3. Click on the ⚙️ **Settings** tab in the top right corner of the page.
+4. Select the **Read tokens** tab from the left-hand menu.
+5. Click on the **Create read token** button.
+
+After creating the read token, you'll see a dialog with the token value.
+**Copy this value and store it securely, it will not be shown again.**
+
+## Using the Read Clients
+
+Logfire provides both synchronous and asynchronous clients to interact with the API.
+These clients are currently experimental, meaning we might introduce breaking changes in the future.
+To use these clients, you can import them from the `experimental` namespace:
+
+```python
+from logfire.experimental.read_client import LogfireAsyncReadClient, LogfireSyncReadClient
+```
+
+### Async Client Example
+
+The `LogfireAsyncReadClient` allows for asynchronous interaction with the Logfire API. Here's an example of how to use it:
```

Comment:
I think it should be called `AsyncLogfireClient` or `AsyncLogfireReadClient`. It's more common in the community.